pysb/simulator/bng.py

```diff
@@ -95,6 +95,9 @@ def run(self, tspan=None, initials=None, param_values=None, n_runs=1,
                                       _run_kwargs=locals()
                                       )
 
+        if cleanup is None:
+            clean = self.cleanup
```

Comment:
I think `clean` is a typo, should be `cleanup`.

Reply:
Gah, indeed! Fixed.